src/style.css

```diff
@@ -1,3 +1,88 @@
+* {
+  /* Скидання параметрів браузера */
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - include an alternative font-family in case the main one doesn't work. For example, 'font-family: Roboto, Arial, sans-serif;'.